Cargo.toml

```diff
@@ -22,10 +22,9 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 
 open = "1.4.0"
-promising-future = "0.2"
 syntect = "4.2"
 horrorshow = "0.8"
-cgraph = { git = "https://github.com/oli-obk/cgraph.git", rev = "7348380f014fe87338cdcd4d2f22cbda771ae21d" }
+cgraph = { git = "https://github.com/oli-obk/cgraph.git", rev = "5e6a6d4527522609772a9a9a1565b74c9bfe1560" }
```

Comment:
I can confirm that https://github.com/oli-obk/cgraph/commit/5e6a6d4527522609772a9a9a1565b74c9bfe1560 is the current master commit.